src/main/java/seedu/address/model/transaction/Transaction.java

```diff
@@ -2,6 +2,8 @@
 
 import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
 
+import java.time.LocalDate;
+
 /**
  * Abstract transaction class
  */
```

Comment:
This class can implement `Comparable<Transaction>` and override `compareTo()` method to aid in Collections.sort()